src/test/lib/ApiTest.cpp

```diff
@@ -3165,956 +3165,947 @@ void QuicTestListenerParam()
 #endif
 }
 
-void QuicTestConnectionParam()
```

Comment:
Why don't we disable compiler optimization as Ben suggested? We could disable compiler optimization for all of our unittests right?

Reply:
Because I'm pretty sure we don't use the LKG of the compiler they refer to because we're undocked. So, I don't know if it'd do anything.

Reply:
Alright, fair enough.